src/cli/aspencli.py

```diff
@@ -294,7 +294,7 @@ def usher():
 def get_link(ctx, sample_ids):
     api_client = ctx.obj["api_client"]
     payload = {"samples": sample_ids, "downstream_consumer": "USHER"}
-    resp = api_client.post("/v2/sequences/getfastaurl", json=payload)
+    resp = api_client.post_with_org("/v2/sequences/getfastaurl", json=payload)
```

Comment:
I updated all the relevant CLI commands to be able to fetch resources with org ID's in the url.